docker/docker-compose.yml

```diff
@@ -1,18 +1,15 @@
 # Note that mongo container is started before Orion, as Orion needs it
 # as dependency and reserve the order has been found problematic in some
 # low resource hosts
-
-# Uncomment the --smallfiles line if you host doesn't have too much free space
 mongo:
```

Comment:
Taking into account the new PR by @LeandroGuillen which address the .md, I think we can work .md changes in that PR. Thus, this becomes a NTC.